### PR TITLE
SCORU originator is not an operator

### PR DESCRIPTION
**Pull request by timothymcmackin**
"Operators" are nodes running in operator mode, not people. Remove the mention of "operator" as the person who originates the rollup.

### CODE REVIEW

File: docs/architecture/smart-rollups.md [hunk 1]
```diff
@@ -91,7 +91,8 @@ Eventually include:
 
 The general flow of a Smart Rollup goes through these phases:
 
-1. Origination: A user called the _rollup operator_ originates the Smart Rollup to layer 1 and one or more users start nodes based on that Smart Rollup to independently verify its operation.
+1. Origination: A user originates the Smart Rollup to layer 1.
```
Comment (mebsout) [inline docs/architecture/smart-rollups.md:94]: The origination is also referred to as the "genesis" of the rollup (genesis level = level of L1 block in which origination operation was included). Not sure this is worth mentioning in this user centric documentation but if it pops up elsewhere maybe you can link back to here.

**Comment by mebsout** [comment]
LGTM.

File: docs/architecture/smart-rollups.md (hunk 1 above)
Comment (timothymcmackin) [reply docs/architecture/smart-rollups.md:94]: Good tip. The Origination section mentions the genesis commitment -- maybe that's enough for now.

Comment (mebsout) [reply docs/architecture/smart-rollups.md:94]: Sounds good :)

**Pull request merged**